### DISCUSSION TITLE
Расшифровка файлов shadow fight 2 и shades.

### DISCUSSION THREAD

**Discussion by VladiSlav0023** [community/community#75006] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Привет, у меня есть вопрос тебе сталкивались зашифрованные файлы которые приходилось расшифровывать ? Если нет то не знаю сможешь расшифровать эти 3 файла. 2 из них практически имеют одну и тоже кодировку (вроде AES), а вот 3 шифровка можно расшифровать через папку lib в apk через загрузочный экран что бы он создавал расшифрованный файл или расшифровал внутри самого файла в виде mdl_data.bin (мне так на видео показывали) только надо знать что надо менять, так как там более 10 файлов so, 
[perks_xml.txt](https://github.com/community/community/files/13317678/perks_xml.txt)
[mdl_armor_light_act1_common_bin_xml.txt](https://github.com/community/community/files/13317679/mdl_armor_light_act1_common_bin_xml.txt)
[mdl_helm_camera_xml.txt](https://github.com/community/community/files/13317680/mdl_helm_camera_xml.txt)

**Comment by Akash1134** [comment]
Hi @VladiSlav0023 , thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.l
